documentation/asciidoc/topics/ref_batch_operations.adoc

```diff
@@ -0,0 +1,121 @@
+[id='batch-operations_{context}']
+= Example Batch Operations
+
+Use these example batch operations as starting points for creating and modifying {brandname} resources with the `Batch` CR.
+
+[NOTE]
+====
+You can pass configuration files to {ispn_operator} only via a `ConfigMap`.
+
+The `ConfigMap` is mounted in {brandname} pods at `/etc/batch` so you must prepend all `--file=` directives with that path.
+====
+
+== Users
+
+Create several {brandname} users and assign them roles with varying levels of permission to access caches and interact with {brandname} resources.
+
+[source,sh,options="nowrap",subs=attributes+]
+----
+echo "connecting"
+connect --username=admin --password=changeme
```

Comment:
Sorry I missed this the first time. The `connect` commands should be removed from all of the batch files.  The Batch CR CLI command already connects and authenticates with the server.